python/cugraph/cugraph/tests/dask/test_mg_bfs.py

```diff
@@ -13,17 +13,17 @@
 
 import cugraph.dask as dcg
 import gc
-import pytest
+# import pytest
 import cugraph
 import dask_cudf
 import cudf
-from cugraph.dask.common.mg_utils import is_single_gpu
+# from cugraph.dask.common.mg_utils import is_single_gpu
 from cugraph.tests.utils import RAPIDS_DATASET_ROOT_DIR_PATH
 
 
-@pytest.mark.skipif(
-    is_single_gpu(), reason="skipping MG testing on Single GPU system"
-)
+# @pytest.mark.skipif(
+#    is_single_gpu(), reason="skipping MG testing on Single GPU system"
+# )
 def test_dask_bfs(dask_client):
     gc.collect()
```

Comment:
optional: I've been using a single module-level teardown function instead (like [this](https://github.com/rapidsai/cugraph/blob/da07cad9adee2df1853e23ab962f53883fb61364/python/cugraph/cugraph/tests/test_bfs.py#L76)) and removing all the individual `gc.collect()` calls.